packages/react/src/autocomplete/Autocomplete.test.tsx

```diff
@@ -54,28 +54,51 @@ describe('Autocomplete', () => {
 	});
 
 	it('can load and clear async options', async () => {
```

Comment:
Here we are checking for multiple things, including that `loadOptions` has been called the correct number of times with the right search term.